doc/articles/getting-started/wizard/extensions.md

```diff
@@ -0,0 +1,80 @@
+---
+uid: Uno.GettingStarted.UsingWizard.Extensions
+---
+
+The settings on the Extensions page let you configure which Uno Extensions should be added to the solution and projects generated by the wizard.
+
+Uno Extensions is a set of Extensions that cover common essentials application core building blocks and are supported on all operating systems Uno Platform targets.  
+The Uno.Extensions follows the Microsoft.Extensions model that creates a host environment where you can register additional dependencies. The registered dependencies are then available throughout the application via the Services (`IServiceProvider`) property on the [`IHost`](https://learn.microsoft.com/en-us/dotnet/api/microsoft.extensions.hosting.ihost) instance.
+
+#### App hosting
+
+Hosting (`IHost`) keeps the central entry point of the app and encapsulates its core building blocks, such as [DI](#dependency-injection), [Logging](#logging), as well as controlling its state and shutdown.  
+Read more about Hosting [here](https://learn.microsoft.com/en-us/dotnet/core/extensions/generic-host).
+
+To learn more about Uno Extensions, visit [this page](xref:Overview.Features).
```

Comment:
the xref seems very generic, is it the name that is used in uno.extensions? If so, we should be prefix them with something like `Uno.Extensions.`

Reply:
It points to [this page](https://github.com/unoplatform/uno.extensions/blob/main/doc/ExtensionsOverview.md).

Reply:
The IDs used in this doc are the ones published.
Newer, less generalized IDs are coming soon. Once merged I'll update this docs.

Reply:
Here's the issue for updating ids - https://github.com/unoplatform/uno.extensions/issues/1556